test/khash_test.erl

```diff
@@ -0,0 +1,443 @@
+-module(khash_test).
+
+-compile([export_all]).
+
+-define(NODEBUG, true).
```

Comment:
NODEBUG [implies](https://github.com/erlang/otp/blob/maint/lib/eunit/include/eunit.hrl#L52-L57) NOASSERT which turns all the assertions into pumpkins.